x-pack/plugins/security_solution/public/management/services/policies/hooks.test.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useGetEndpointSecurityPackage } from './hooks';
+import { HttpFetchError, HttpSetup } from 'kibana/public';
+import { getFakeHttpService, renderQuery } from '../../hooks/artifacts/test_utils';
```

Comment:
@dasansol92  i borrowed these functions from the test util file you created. Do you think I should move just these 2 items to another file in common or should i move the entire file up?

Reply:
Awesome! If this is something you can reuse at all let's move it in an upper level 🙂